wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -42,9 +42,27 @@ class SchedulerState(Enum):
 
 
 class RunState(Enum):
-    ALIVE = 0
-    DEAD = 1
-    UNKNOWN = 2
+    RUNNING = "running", "alive"
+    PENDING = "pending", "alive"
+    PREEMPTING = "preempting", "alive"
+    CRASHED = "crashed", "dead"
+    FAILED = "failed", "dead"
+    KILLED = "killed", "dead"
+    FINISHED = "finished", "dead"
+    PREEMPTED = "preempted", "dead"
+    UNKNOWN = "unknown", "alive"  # TODO(gst): dead?
```

Comment:
When do we get an unknown state?

Reply:
State is unknown when we hit the api.get_run_state and it either errors or returns nonsense

Reply:
What nonsense is returned? If an error is thrown, we can either assume the run does not exist (which I think indicates a user deleted it?) or there's something more sinister going on, like an auth issue, which would be dead to the scheduler no?